### PR TITLE
Add link to MeetYourNextMP

### PR DESCRIPTION
**Pull request by jarofgreen**
I couldn't get the vagrant install working so it's not tested. Sorry!

### CODE REVIEW

File: candidates/views/constituencies.py [hunk 1]
```diff
@@ -59,6 +59,8 @@ def get_context_data(self, **kwargs):
         context['electionleaflets_url'] = \
             get_electionleaflets_url(context['constituency_name'])
 
+        context['meetyournextmp_url'] = u'https://meetyournextmp.com/linktoseat.html?mapitid=' . mapit_area_id
```
Comment (andylolz) [inline candidates/views/constituencies.py]: string concatenation in python is done with a `+`. But it’s probably preferable to do:

```
context['meetyournextmp_url'] = u'https://meetyournextmp.com/linktoseat.html?mapitid=%s' % mapit_area_id
```

Comment (andylolz) [inline candidates/views/constituencies.py]: …or even better (as @tfgg points out):

```
context['meetyournextmp_url'] = u'https://meetyournextmp.com/linktoseat.html?mapitid={}'.format(mapit_area_id)
```

**Comment by jarofgreen** [comment]
Fixed in https://github.com/mysociety/yournextmp-popit/pull/227 - please kill this

**Comment by mhl** [comment]
I'm closing this since it's superseded by #227

**Pull request closed without merging**